api/hiveextension/v1beta1/agentclusterinstall_types.go

```diff
@@ -138,6 +138,10 @@ type AgentClusterInstallSpec struct {
 	// IngressVIP is the virtual IP used for cluster ingress traffic.
 	// +optional
 	IngressVIP string `json:"ingressVIP,omitempty"`
+
+	// Cluster installation will not start as long as HoldInstallation is set to true.
```

Comment:
I think it's worth adding some detail that validation and such will still happen. We don't want this to be confused with a generic "Pause" button that will stop progress toward installation; it just adds a gate at a specific part of the process, and we should describe exactly where that is.

Reply:
That's an improvement. I suggest being explicit about when the behavior does have an effect. Also I noticed that this needs to be re-phrased to start with the name of the field, which is [a golang doc convention that supports good doc generation](https://blog.golang.org/godoc). Maybe something like this:

`HoldInstallation will prevent installation from happening when true. Inspection and validation will proceed as usual, but once the RequirementsMet condition is true, installation will not begin until this field is set to false.`